sources/lib/util/options.ml

```diff
@@ -34,6 +34,7 @@ exception Error of bool * string
 (* Declaration of all the options as refs with default values *)
 
 type model = MNone | MDefault | MAll | MComplete
+type in_out_format = Native | Smtlib | Why3 (* | SZS *)
```

Comment:
We should keep two different types: `input_format` and `output_format`.

Reply:
Since you don't want disambiguation such as `INative`, etc, and you style want two types you have this kind of errors :
`This expression has type output_format but an expression was expected of type input_format`

Reply:
When you have these errors, you can add a few type hints here and there (usually, specifying the type of a function argument or of a match construct is more than enough).